setup.py

```diff
@@ -52,7 +50,9 @@
     'develop', 'release', 'bdist_egg', 'bdist_rpm',
     'bdist_wininst', 'install_egg_info', 'build_sphinx',
     'egg_info', 'easy_install', 'upload', 'bdist_wheel',
-    '--single-version-externally-managed',
+    '--single-version-externally-managed', 'install', 'build', 'build_ext',
```

Comment:
I share your dislike for ``setuptools``, it's awful. And I think you can still get away without it. For numpy/scipy that wasn't possible, because they add the git hash to the dev version number and that results in completely broken installs without ``setuptools`` because ``distutils`` doesn't support PEP440 and ``pip`` enforces it.  So I'm happy to change this.

> I fear that I fail to see the benefit that it brings us.

Consistency. If reverting this, then I do think that ``develop`` and ``--single-version-externally-managed`` should also be removed, because it's bad to get different behavior depending on which particular build command you use. For example ``setup.py develop`` and ``setup.py build_ext -i`` should basically be identical for the build part, and now only the former does ``import setuptools``.